src/ipyradiant/visualization/datashader_vis.py

```diff
@@ -0,0 +1,135 @@
+import traitlets as T

```

Comment:
Tiny nitpick, is there a reason that this file is `datashader_vis.py` but cytoscape is `cytoscape.py`?
Since they are in our `visualization` module it may just be cleaner to call this file `datashader.py`.

Reply:
i did `datashader_vis.py` since `datashader` is already a package name.. but i can change it back if necessary/desired

Reply:
Nope, that makes sense. We can come up with a more generic pattern once we have more visualizations to compare.